upf/test/test_upf.py

```diff
@@ -147,6 +147,9 @@ def ie_fteid_ch(self, chid):
     def ie_ue_ip_address(self, SD=0):
         return IE_UE_IP_Address(ipv4=self.ue_ip, V4=1, SD=SD)
 
+    def ie_ue_ip_address2(self, SD=0):
```

Comment:
Remove this func, just add an optional `ip=None` argument to `ie_ue_ip_address()`